templates/procuration_manager/request.html.twig

```diff
@@ -17,12 +17,12 @@
                 Désassocier de {{ request.foundProxy.firstNames }} {{ request.foundProxy.lastName }}
             </a>
         {% elseif request.processed %}
-            <a href="{{ path('app_procuration_manager_request_transform', {'id': request.id, 'action': 'detraiter', 'token': csrfToken}) }}"
+            <a href="{{ path('app_procuration_manager_request_transform', {'id': request.id, 'action': 'detraiter', 'token': csrf_token('request_action')}) }}"
```

Comment:
Simples quotes in array keys are useless

Reply:
I know this is not my code and I didn't want to change it everywhere... fixed in https://github.com/EnMarche/en-marche.fr/pull/1924/commits/c4be2d0911d19fb5148198bcf70cb395bd8b318f.